### DISCUSSION TITLE
Applying transparency to mantine colors

### DISCUSSION THREAD

**Discussion by j-tag** [mantinedev/mantine#5311] [Q&A] [answered]
I want to apply transparency only to background color of a div element, so then I can apply blur filter behind it. I've used following CSS:

```
.blur {
    background-color: rgba(var(--mantine-color-body), .66);
    backdrop-filter: blur(5px);
}
```

I wanted to apply transparency to `--mantine-color-body` but it doesn't work and the background color becomes totally transparent. I can't use `opacity` because it'll apply transparency to all of div children also which I don't want that to happen.
If I use hardcoded colors like ` rgba(100, 100, 100, .66)` it'll work as expected.

Any suggestion would be appreciated, thanks!

**Comment by rtivital** [comment, marked as answer]
You cannot change the opacity of CSS variables. Use color values instead.

**Reply by j-tag** [reply]
Thanks for your reply Vitaly. Yea I see.

So I wanted to use different colors for dark and light themes and I think this is my best bet:

```
.blur {
    background-color: light-dark(rgba(200, 200, 200, .66), rgba(26, 27, 30, .66));
    backdrop-filter: blur(5px);
}
```

**Reply by kinoli** [reply]
Can you use theme colors and apply alpha on those. MaterialUI has an alpha function allowing you to pass hex values and a percentage into it. Does Mantine have anything similar? Such as `border-bottom: 1px solid ${alpha(theme.palette[color].dark, 0.2)};`

**Reply by rtivital** [reply]
https://mantine.dev/styles/color-functions/#alpha

**Reply by kinoli** [reply]
Was that added in v7? I don't see it in v6.

**Reply by rtivital** [reply]
Yes, it was

**Reply by JovanaBozhinovska** [reply]
is there a way to use this in modules.css?

**Reply by rtivital** [reply]
https://mantine.dev/styles/postcss-preset/#alpha-function
